news/templates/news/user_news.html

```diff
@@ -1,46 +1,38 @@
 {% load i18n %}
 {% load news %}
 
-{% for item in alerts %}
-<div class="alert alert-{{ item.alert }}">
-  <h4>
-    {{ item.title|safe }}
-    {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>{% endif %}
-    {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-  </h4>
-  <div>
-    {{ item.body|safe }}
-  </div>
-</div>
-{% endfor %}
 
 {% if news %}
 <div class="panel panel-default news-panel">
   <div class="panel-heading">
-    <h4 class="panel-title">{% trans "Course news" %}</h4>
+    <h3 class="panel-title">{% trans "Course news" %}</h3>
   </div>
   <div class="list-group">
     {% for item in news %}
-    <a class="list-group-item{% if item.collapsed %} collapsed{% endif %}" href="#news{{ item.id }}" data-toggle="collapse" role="button" aria-controls="news{{ item.id }}" aria-expanded="{% if item.collapsed %}false{% else %}true{% endif %}">
+    <div class="{% if item.pin %}pinned-{% endif %}list-group-item">
       <h4 class="list-group-item-heading">
-        <span class="caret"></span>
-        {{ item.title|safe }} <small>{{ item.publish }}</small>
-        {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears later" %}</span></small>{% endif %}
-        {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
+        {{ item.title|safe }}
+        {% if not item|is_published:now %}
+          <small class="future-instance">{% trans "Published on" %}: {{ item.publish }}</small>
+        {% else %}
+          <small class="current-instance">{{ item.publish }}</small>
+        {% endif %}
+        {% if is_course_staff %}<small><span class="label label-primary">{{ item.audience|news_audience }}</span></small>{% endif %}
+        {% if is_course_staff and item.pin %}<small><span class="glyphicon glyphicon-pushpin"></span></small>{% endif %}
```

Comment:
Should the glyph be visible for all users? I think it could be nice for students too.

Reply:
I thought about that but since you actually said that there is no need to emphasize the pinning for students, I left the pin out for them :D But I added it now for students as well, I think it is nice for them too :)